client/reporter.js

```diff
@@ -194,9 +146,6 @@ Reporter.prototype.done = function (err) {
 
   // add coverage tab content
   if (window.__coverage__) {
-    var coverageTab = document.getElementById('code-coverage-tab')
-    coverageTab.innerHTML = '<iframe frameborder="0" src="/airtap/coverage"></iframe>'
```

Comment:
@vweevers A bit unsure of this but I'm guessing it's not needed anymore. Also, what actually defines the `__coverage__` global?

Reply:
Yeah, we don't need that. As for the `__coverage__` global, that's populated by the `browserify-istanbul` transform.